src/main/java/com/klodnicki/Bike/v2/rest/controller/AdminChargingStationController.java

```diff
@@ -0,0 +1,36 @@
+package com.klodnicki.Bike.v2.rest.controller;
+
+import com.klodnicki.Bike.v2.DTO.station.ListStationsForAdminResponseDTO;
+import com.klodnicki.Bike.v2.DTO.station.StationForAdminResponseDTO;
+import com.klodnicki.Bike.v2.model.entity.ChargingStation;
+import com.klodnicki.Bike.v2.service.ChargingStationService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/admin/stations")
+@AllArgsConstructor
+public class AdminChargingStationController {
+
+    private final ChargingStationService chargingStationService;
```

Comment:
Interface should be used here instead of concrete class

Reply:
yes!!! you are absolutely right! thanks!